examples/models/llama2/CMakeLists.txt

```diff
@@ -0,0 +1,253 @@
+endif()
+
+if(NOT FLATC_EXECUTABLE)
+  set(FLATC_EXECUTABLE flatc)
+endif()
+
+# Source root directory for executorch.
+if(NOT EXECUTORCH_ROOT)
+  set(EXECUTORCH_ROOT ${CMAKE_CURRENT_SOURCE_DIR}/../../..)
+endif()
+
+# Source root directory for pytorch.
+if(NOT TORCH_ROOT)
+  set(TORCH_ROOT ${EXECUTORCH_ROOT}/third-party/pytorch)
+endif()
+
+add_compile_options("-Wall" "-Werror")
+
+include(${EXECUTORCH_ROOT}/build/Utils.cmake)
+
+set(_common_compile_options -Wno-deprecated-declarations -fPIC)
+
+# Let files say "include <executorch/path/to/header.h>".
+set(_common_include_directories ${EXECUTORCH_ROOT}/..)
+
+# Find prebuilt libraries. executorch package should contain
+# portable_ops_lib, etdump, bundled_program.
+find_package(executorch CONFIG REQUIRED)
+target_include_directories(executorch INTERFACE ${_common_include_directories})
+target_compile_options(executorch INTERFACE -DET_EVENT_TRACER_ENABLED)
+
+find_package(
+  gflags REQUIRED PATHS ${CMAKE_CURRENT_BINARY_DIR}/../../../third-party
+)
+
+# portable_ops_lib
+include(${EXECUTORCH_ROOT}/build/Utils.cmake)
+include(${EXECUTORCH_ROOT}/build/Codegen.cmake)
+gen_selected_ops("" "" "ON")
+generate_bindings_for_kernels(
+  ${EXECUTORCH_ROOT}/kernels/portable/functions.yaml ""
+)
+gen_operators_lib("portable_ops_lib" portable_kernels executorch)
+
+set(llama_runner_libs "-framework Foundation")
+
+#
+# The `_<target>_srcs` lists are defined by including ${EXECUTORCH_SRCS_FILE}.
+#
+set(
+  EXECUTORCH_SRCS_FILE
+  "${CMAKE_CURRENT_BINARY_DIR}/../../../executorch_srcs.cmake"
+)
+
+extract_sources(${EXECUTORCH_SRCS_FILE})
+
+
+add_library(extension_module "${CMAKE_CURRENT_SOURCE_DIR}/../../../extension/module/module.cpp")
+target_include_directories(extension_module PUBLIC ${EXECUTORCH_ROOT}/..)
+
+#add_library(extension_data_loader "${CMAKE_CURRENT_SOURCE_DIR}/../../../extension/data_loader/mmap_data_loader.cpp")
+#target_link_libraries(extension_data_loader executorch)
+
+
+# Install libraries
+install(
+  TARGETS extension_module
+  DESTINATION ${CMAKE_BINARY_DIR}/lib
+  INCLUDES
+  DESTINATION ${_common_include_directories})
+
+
+set(llama_schema_headers ${CMAKE_BINARY_DIR}/../../../schema/include/)
+include(${EXECUTORCH_SRCS_FILE})
+target_include_directories(
+  bundled_program
+  INTERFACE
+  ${CMAKE_CURRENT_BINARY_DIR}/../../../sdk/include
+  ${CMAKE_CURRENT_BINARY_DIR}/../../../sdk/bundled_program
+  ${EXECUTORCH_ROOT}/third-party/flatbuffers/include
+  ${EXECUTORCH_ROOT}/third-party/flatcc/include
+  ${llama_schema_headers}
+)
+
+function(define_common_targets)
+    foreach(aten IN ITEMS OFF) # ON OFF
+        if(aten)
+            set(aten_suffix "_aten")
+        else()
+            set(aten_suffix "")
+        endif()
+
+        message(">>>>>> CMAKE_CURRENT_SOURCE_DIR: ${CMAKE_CURRENT_SOURCE_DIR}")
+
+        set(EXPORTED_HEADERS_RUNNER
+            "${CMAKE_CURRENT_SOURCE_DIR}/runner/runner.h"
+            "${CMAKE_CURRENT_SOURCE_DIR}/runner/util.h"
+        )
```

Comment:
We don't need this because we include the whole executorch/ directory like here https://github.com/pytorch/executorch/blob/main/examples/selective_build/CMakeLists.txt#L45

Reply:
without this i got en error
```
Built target flatcc_project
make: *** [all] Error 2
```